internal/generate/create.go

```diff
@@ -23,11 +23,11 @@ func generatorCreateCommand() *command {
 	c := &command{
 		name:  "create",
 		short: "Create a new client library",
-		usage: "generator create [arguments]",
+		usage: "create [arguments]",
```

Comment:
I suspect this could be inferred from "name" - given that we always expect to basically parse flags in a consistent way. (i.e. I'd remove this entirely, and let anything that uses `usage` at the moment use `name` instead.)

It's only a tiny bit of redundancy though.

Reply:
yup sounds good - removed